include/test/battle.h

```diff
@@ -584,6 +602,42 @@ struct BattlerTurn
     struct TurnRNG rng;
 };
 
+struct ExpectedAiAction
```

Comment:
I'd call this `ExpectedAIAction` with a capital I. I think that's usually how we do our names (although admittedly it's not 100% consistent across the codebase). There's quite a few other examples of `Ai` instead of `AI`, I'd be happy to fix them all if it sounds like too boring to do :sweat_smile: